src/utils/v2APIBreak.ts

```diff
@@ -229,7 +229,7 @@ export const v2APIBreak = () => {
   }
 
   // API break function is called before the version31 2.3.0 changes, scripts are still an array.
```

Comment:
This comment can be removed now that 2.3 migration code is actually called first, just noting it so we can remove at some point. Will go ahead and merge this -> cherry pick over to stable -> rerelease new hotfix

Reply:
Whoops, missed that one